addons/calendar/data/mail_template_data.xml

```diff
@@ -132,121 +127,117 @@
     <t t-set="is_online" t-value="'appointment_type_id' in object.event_id and object.event_id.appointment_type_id"/>
     <t t-set="customer" t-value="object.event_id.find_partner_customer()"/>
     <t t-set="target_responsible" t-value="object.partner_id == object.event_id.partner_id"/>
     <t t-set="target_customer" t-value="object.partner_id == customer"/>
-     <t t-set="recurrent" t-value="object.recurrence_id and not ctx.get('calendar_template_ignore_recurrence')"/>
+    <t t-set="recurrent" t-value="object.recurrence_id and not ctx.get('calendar_template_ignore_recurrence')"/>
+    <t t-set="organizer" t-value="object.event_id.user_id"/>
 
-    <p>
-        Hello <t t-out="object.common_name or ''">Ready Mat</t>,<br/><br/>
+    <h1 style="font-size: 18px; font-weight: bold; color: #374151; margin-bottom: 28px;">
+        Date Updated
+        <img src="/calendar/static/src/img/sync.png" style="display: inline-block; border:0; outline:none; width: 24px; height: 24px; margin-left: 10px; text-decoration:none; vertical-align: bottom;" />
+    </h1>
+    <p style="margin: 0;">
+        <span style="display: block; margin-bottom: 8px;">Hello <t t-out="object.common_name or ''">Ready Mat</t>,</span>
         <t t-if="is_online and target_responsible">
             <t t-if="customer">
-                The date of your appointment with <t t-out="customer.name or ''">Jesse Brown</t> has been updated.
+                Your appointment with <t t-out="customer.name or ''">Jesse Brown</t> has been updated:
             </t>
             <t t-else="">
-                Your appointment has been updated.
+                Your appointment has been updated:
             </t>
-            The appointment <strong t-out="object.event_id.appointment_type_id.name or ''">Schedule a Demo</strong> is now scheduled for
-            <t t-out="object.event_id.get_display_time_tz(tz=object.partner_id.tz) or ''">05/04/2021 at (11:00:00 To 11:30:00) (Europe/Brussels)</t>
+            <strong t-out="object.event_id.appointment_type_id.name or ''">Schedule a Demo</strong> is now scheduled for
+            <t t-out="object.event_id.get_display_time_tz(tz=object.partner_id.tz) or ''">05/04/2021 at (11:00:00 To 11:30:00) (Europe/Brussels)</t>.
         </t>
         <t t-elif="is_online and target_customer">
-            The date of your appointment with <t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> has been updated.
-            The appointment <strong t-out="object.event_id.appointment_type_id.name or ''"></strong> is now scheduled for
-            <t t-out="object.event_id.get_display_time_tz(tz=object.partner_id.tz) or ''">05/04/2021 at (11:00:00 To 11:30:00) (Europe/Brussels)</t>.
+            Your appointment <strong t-out="object.event_id.appointment_type_id.name or ''"></strong> <t t-if="organizer">with </t><t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t>
```

Comment:
can we reuse `organizer` for readability? `organizer.partner_id.name` same below